### PR TITLE
Bump bdk to version 0.11

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

**Review by bonomat — APPROVED** [review]

File: cfd_protocol/tests/dlc_protocol.rs [hunk 1]
```diff
@@ -417,7 +417,13 @@ fn build_wallet<R>(
 where
     R: RngCore + CryptoRng,
 {
-    use bdk::{populate_test_db, testutils};
+    // TODO: Consider upstreaming these imports to be included in the macro.
```
Comment (luckysori) [inline cfd_protocol/tests/dlc_protocol.rs:420]: :rofl: I already did this: https://github.com/bitcoindevkit/bdk/pull/428

**Review by luckysori — COMMENTED** [review]
Apparently `bdk:0.11` doesn't include https://github.com/bitcoindevkit/bdk/pull/428. Shall we just wait?

File: cfd_protocol/tests/dlc_protocol.rs (hunk 1 above)
Comment (thomaseizinger) [reply cfd_protocol/tests/dlc_protocol.rs:420]: Lucas, reading my thoughts before I have them, amazing!

**Comment by thomaseizinger** [comment]
Hang on, are we depending on a rev that DOES include https://github.com/bitcoindevkit/bdk/pull/428 already? I am a moron then!

Ditch the upgrade haha.

**Pull request closed without merging**